src/lib.rs

```diff
@@ -19,35 +19,91 @@ extern crate proc_macro;
 mod pb_convert;
 
 use proc_macro::TokenStream;
-use syn::{Attribute, Meta, MetaList, MetaNameValue, NestedMeta};
+use syn::{Attribute, NestedMeta};
 
-// Derive attribute names, used as
-// `#[protobuf_convert( [ ATTRIBUTE_NAME = ATTRIBUTE_VALUE or ATTRIBUTE_NAME ],* )]`
-const PB_CONVERT_ATTRIBUTE: &str = "pb";
-const SERDE_PB_CONVERT_ATTRIBUTE: &str = "serde_pb_convert";
+const PROTOBUF_CONVERT_ATTRIBUTE: &str = "protobuf_convert";
+const PROTOBUF_CONVERT_SKIP_ATTRIBUTE: &str = "skip";
+const DEFAULT_ONEOF_FIELD_NAME: &str = "message";
```

Comment:
In witnet we use `kind` as the default field name, but it can be changed to `message` if you think that's a better name.

Reply:
We have only a few places where we uses enums, therefore we can use `kind` without pain.